README.md

```diff
@@ -199,97 +135,107 @@ run ActionController::Dispatcher.new
-# If you don't set REDIS_URL, remove that to use default localhost
+* <a href="#installation">Install Coverband</a>
+* Start your app and hit a few endpoints
+* Validate data collection and code coverage with  `rake coverband:coverage`
+* If you see app startup and recent visits showing, setup is correct
+* I generally configure the mountable web endpoint to [view the data via the web-app](https://github.com/danmayer/coverband#viewing--hosting-s3-coverband-results-in-app)
+* After Coverband has been verified to be working on production, I let it run for a few weeks.
+* Then I view the report and start submitting PRs for the team to review delete large related sets of code that no longer are in use.
 
-# Make some code so we can look at the coverage
-rails generate scaffold blogs
-rake db:migrate
+# Advanced Usage
 
-# open Rakefile, add lines
-require 'coverband'
-Coverband.configure
-require 'coverband/utils/tasks'
+### Example apps
 
-# verify rake
-rake -T coverband
+- [Rails 5.2.x App](https://github.com/danmayer/coverage_demo)
+- [Sinatra app](https://github.com/danmayer/churn-site)
+- [Non Rack Ruby app](https://github.com/danmayer/coverband_examples)
 
-# configure config/application.rb
-# copy lines from readme
+### Example Output
 
-# start up your app
-rake coverband:coverage
-# view boot up coverage
+Since Coverband is [Simplecov](https://github.com/colszowka/simplecov) output compatible it should work with any of the `SimpleCov::Formatter`'s available. The output below is produced using the default Simplecov HTML formatter.
 
-rails s
+Index Page
+![image](https://raw.github.com/danmayer/coverband/master/docs/coverband_index.png)
 
-open http://localhost:3000/blogs
+Details on a example Sinatra app
+![image](https://raw.github.com/danmayer/coverband/master/docs/coverband_details.png)
 
-# click around some to trigger usage
+### Advanced Coverband Config
 
-# view updated coverage
-rake coverband:coverage
-```
+You may need to configure Coverband you can either do that passing in all configuration options to `Coverband.configure` in block format, or a simpler style is to call `Coverband.configure` with no params which will load `config/coverband.rb` expecting it to configure the app correctly.
 
-# Usage
+* See [lib/coverband/configuration.rb](https://github.com/danmayer/coverband/blob/master/lib/coverband/configuration.rb) for all options
+* I strongly recommend setting up the S3 report adapter, which can't be automatically configured
+* By default Coverband will try to stored data to Redis
+	* Redis endpoint is looked for in this order: `ENV['COVERBAND_REDIS_URL']`, `ENV['REDIS_URL']`, or `localhost`
 
-### Example apps
+ Below is an example config file for a Rails 5 app:
 
-- [Rails 5.2.x App](https://github.com/danmayer/coverage_demo)
-- [Sinatra app](https://github.com/danmayer/churn-site)
-- [Non Rack Ruby app](https://github.com/danmayer/coverband_examples)
+```ruby
+#config/coverband.rb
+Coverband.configure do |config|
```

Comment:
Think we should mention that we can set the environment variables or configure through coverband with config.s3_access_key_id and config.s3_secret_access_key?

Reply:
good call updated docs around this as well